src/main/java/frc/robot/HardwareMap.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.XboxController;
+
+public class HardwareMap {
+
+    public VictorSPX intakeController;
+    public VictorSPX armController;
+    public XboxController operatorJoystick;
```

Comment:
pretty sure these can be static and final

Reply:
probably but we will address that today as part of a larger discussion on hardware map